plugins/woocommerce-admin/client/customize-store/transitional/index.tsx

```diff
@@ -260,6 +252,7 @@ export const Transitional = ( {
 									</p>
 									<Button
 										variant="link"
+										href={ `${ ADMIN_URL }admin.php?page=wc-admin` }
```

Comment:
This line is most likely not needed. "Back to Home" button/link triggers an event that causes a state transition rather than navigation. I'm consulting colleagues about the next steps regarding this, let's wait for their response.

Reply:
@kmanijak Thanks for the update, I will definitely wait for the feedback on this.